include/common_table.h

```diff
@@ -38,7 +38,11 @@ typedef struct CommonTable
 } Table;
 
 typedef enum
-{ KEY, SERVER, TEST } TableType;
+{ 
+  KEY,      /**< Table to store KEKs*/
```

Comment:
The channel key table is very similar to table of type ```key``` but has some key differences so that I recommend that we do not re-use the common functionality. First, the channel key table does not have an key_id but is determined by the index location of the key for pairing. Secondly, the channel key table does not grow and shrink with the number of channels so the full table is allocated on initialization. These differences make incorporation of the channel key table into the common functionality seem more difficult then having its own code functions.